wallet/wallet.go

```diff
@@ -1343,6 +1357,62 @@ b:
 	return &msg, nil
 }
 
+func txMultiSign(redeemTx types.Transaction, network string) (string, error) {
+
+
+	var param *chaincfg.Params
+	switch network {
+	case "mainnet":
+		param = &chaincfg.MainNetParams
+	case "testnet":
+		param = &chaincfg.TestNetParams
+	case "privnet":
+		param = &chaincfg.PrivNetParams
+	case "mixnet":
+		param = &chaincfg.MixNetParams
+	}
+
+
+
+	var sigScripts [][]byte
+	for i := range redeemTx.TxIn {
+		privkeyByte := redeemTx.TxIn[i].SignScript
```

Comment:
1.) should not cache the private-key in `txin.sign_script` of input parameter `redeemTx`. 
2.) It's not multi-sign actually, it leads to a misunderstanding of the real purpose. the use-case is strange. Please give more documentation for the actual purpose and do a better renaming & refactoring according to 1.)